app/src/main/java/org/openobservatory/ooniprobe/common/DateAdapter.java

```diff
@@ -0,0 +1,37 @@
+package org.openobservatory.ooniprobe.common;
+
+import com.google.gson.TypeAdapter;
+import com.google.gson.internal.bind.util.ISO8601Utils;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonToken;
+import com.google.gson.stream.JsonWriter;
+
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.ParsePosition;
+import java.util.Date;
+
+class DateAdapter extends TypeAdapter<Date> {
+	@Override public void write(JsonWriter out, Date value) throws IOException {
+		if (value == null)
```

Comment:
Do you mind adding `{` and `}` here? It's more robust to refactoring

Reply:
done